src/Core/src/Platform/Android/MauiAppCompatActivity.cs

```diff
@@ -29,44 +29,20 @@ protected override void OnCreate(Bundle? savedInstanceState)
 				throw new InvalidOperationException($"The {nameof(IServiceProvider)} instance was not found.");
 
 			var mauiContext = new MauiContext(services, this);
-
 			var state = new ActivationState(mauiContext, savedInstanceState);
 			var window = mauiApp.CreateWindow(state);
+			SetContentView(window.View.ToNative(mauiContext));
```

Comment:
just set the root to the page. It's confusing and unexpected to do anything else.

If someone wants a view at the root they can do that themselves inside the IView that shouldn't be done here

Reply:
Azure Pipelines could not run because the pipeline triggers exclude this branch/path.